### ISSUE TITLE
pipeline.index M.randomUUID is not a function TypeError: M.randomUUID is not a function

### ISSUE THREAD

**Issue by mmm8955405** [ConduitIO/conduit#1207]
### Bug description

Uncaught (in promise) TypeError: M.randomUUID is not a function

### Steps to reproduce

![1](https://github.com/ConduitIO/conduit/assets/5687907/28833d03-b225-44f5-ac84-535dc469f019)

### Version

last

**Comment by hariso** [comment]
@mmm8955405 Thanks for reporting this!

IIUC, you found this in the browser logs, and I think it should be investigated. However, what issue are you facing because of this, i.e. what part of Conduit's functionality is broken because of that?

**Comment by simonl2002** [comment]
@mmm8955405 wanted to check in here to see if you could provide us any additional information about this error.

**Comment by mmm8955405** [comment]
![1](https://github.com/ConduitIO/conduit/assets/5687907/d8b592c1-586e-4c67-b057-5961f64e41c0)
I'm very sorry, I only saw your message now. I believe this issue occurs above v0.72, and v0.62 is normal. Appears when the create pipeline button is clicked. What more information do you need to know?

**Comment by hariso** [comment]
@mmm8955405 Despite that, you're able to create a pipeline, is that correct?

**Comment by mmm8955405** [comment]
> @mmm8955405 Despite that, you're able to create a pipeline, is that correct?

Unable to create pipeline

**Comment by mmm8955405** [comment]
![1](https://github.com/ConduitIO/conduit/assets/5687907/6a0fd696-75c6-4883-87eb-fa5acb33919b)
connnduit docker image v0.72

**Comment by lovromazgon** [comment]
Maybe this is an issue related to a specific browser. @mmm8955405 can you tell us what browser and version you're using?

**Comment by mmm8955405** [comment]
Google Chrome 117.0.5938.150 （64 位）

**Comment by mmm8955405** [comment]
Very strange

**Comment by mmm8955405** [comment]
I am running this image. Is it a permission issue? I will continue to check

**Comment by mmm8955405** [comment]
I have found this issue, which may be related to the CDN of JS. After I use the proxy, it works normally

**Comment by hariso** [comment]
@mmm8955405 What if you run it locally?

**Comment by mmm8955405** [comment]
I would like to temporarily close this one

**Comment by MichalisDBA** [comment]
Hello. Same problem. I am running AlmaLinux 9.4. I downloaded the latest .rpm file `conduit_0.10.2_Linux_x86_64.rpm` and install it with `rpm -ivh conduit_0.10.2_Linux_x86_64.rpm`

i started the conduit by simply run `cunduit`.

The server started and open the webpage. Tried to press `Create pipeline` but it does not work. In console there are error messages

![image](https://github.com/ConduitIO/conduit/assets/68512501/f10a6f6b-01c9-4add-b51c-b94fc7e0ea87)
